src/Juvix/Compiler/Backend/Geb/Translation/FromCore.hs

```diff
@@ -120,7 +120,7 @@ fromCore tab = case tab ^. Core.infoMain of
         nodeType = convertType (Info.getNodeType node)
 
     convertNode :: Core.Node -> Trans Morphism
-    convertNode = \case
+    convertNode node = case node of
```

Comment:
If you're not using node, please leave it as it was.

Reply:
Done